src/implementations/chrono.rs

```diff
@@ -1,6 +1,8 @@
+#![cfg(feature = "chrono")]
```

Comment:
Shouldn't we put these on module definitions instead? What happens when the feature is not enabled? Won't the `mod` statement break?

Reply:
It doesn't appear to @rushmorem ... !